adapters/coap-vertx-base/src/main/java/org/eclipse/hono/adapter/coap/CoapContext.java

```diff
@@ -320,6 +331,18 @@ public Integer getCommandResponseStatus() {
         return getIntegerQueryParameter(Constants.HEADER_COMMAND_RESPONSE_STATUS);
     }
 
+    /**
+     * Check, if request is sent from a device, which version supports piggybacked response.
+     *
+     * Overwrite a tenant specific disabled {@link CoapConstants#TIMEOUT_TO_ACK}.
+     *
+     * @return {@code true}, if device version supports piggybacked responses, {@code false}, for devices with "legacy
+     *         version", which doesn't support that.
+     */
+    public boolean isSupportingPiggyBacked() {
```

Comment:
given that this is supposed to be an undocumented feature we should probably keep this method private

Reply:
OK

Reply:
Done.